test/e2e/registration_service_test.go

```diff
@@ -486,7 +486,7 @@ func (s *registrationServiceTestSuite) TestPhoneVerification() {
 	require.False(s.T(), mpStatus["verificationRequired"].(bool))
 
 	// Now approve the usersignup.
-	userSignup.Spec.Approved = true
+	states.SetApproved(userSignup, true)
```

Comment:
Maybe worth leaving at least one test which uses the old API to verify the migration code works?

Reply:
I think the unit tests are sufficient for verifying the migration, don't you?

Reply:
Covering it by e2e would be even better if it's easy to do. So, if we simply leaving one test which uses the old API as is, then we might get a migration thing covered by our e2e very cheap, right?

Reply:
No problem, I've reverted to the original code here in order to test the migration.